sdk/typescript/src/types/events.ts

```diff
@@ -129,9 +129,15 @@ export type SuiEventFilter =
   | { And: [SuiEventFilter, SuiEventFilter] }
   | { Or: [SuiEventFilter, SuiEventFilter] };
 
+export type SuiEventId = {
+  tx_seq_num: number;
```

Comment:
These will need to be replaced with a `bignum` to prevent possible overflow from the `i64` type, but I will take care of that in another or

Reply:
@stella3d curious what is the range of number compared to i64?